microsoft-edge/webview2/reference/javascript/webviewmessageevent.yml

```diff
@@ -0,0 +1,44 @@
+### YamlMime:TSType
+name: WebViewMessageEvent
+uid: 'WebView2Script!WebViewMessageEvent:class'
+package: WebView2Script!
+fullName: WebViewMessageEvent
+summary: ''
+remarks: ''
+isPreview: false
+isDeprecated: false
+type: interface
+properties:
+  - name: additionalObjects
+    uid: 'WebView2Script!WebViewMessageEvent#additionalObjects:member'
+    package: WebView2Script!
+    fullName: additionalObjects
+    summary: >-
+      Additional DOM objects that are sent via PostJSONMessageWithAdditionalObjects. These objects can be of following
+      types:
+      
+      DOM | Win32
+      ------- | ------------
+      [FileSystemFileHandle](https://developer.mozilla.org/docs/Web/API/FileSystemFileHandle) | ICoreWebView2FileSystemHandle (kind File)

```

Comment:
Missing description: empty cell for webviewmessageevent in bottom of table at
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/?branch=pr-en-us-3110#classes